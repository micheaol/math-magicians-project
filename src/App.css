@import url('https://fonts.googleapis.com/css2?family=Mohave:ital,wght@1,300&family=Roboto+Condensed:wght@700&family=Roboto:wght@300&display=swap');
@import url('https://fonts.googleapis.com/css2?family=Mohave:ital,wght@1,300&family=Outfit:wght@300&display=swap');

.home-wrapper {
  text-align: start;
  padding: 5%;
  font-family: 'Outfit', sans-serif;
}

.quote-wrapper {
  text-align: center;
  margin-top: 15%;
  font-family: 'Roboto Condensed', sans-serif;
}
