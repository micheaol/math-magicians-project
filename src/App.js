import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import './App.css';
import Calculator from './components/Calculator';
import Home from './components/Home';
import Navbar from './components/Navbar';
import Quote from './components/Quote';

function App() {
  return (
    <Router>
      <div>
        <Navbar />
        <Switch>
          <Route exact path="/">
            <Home />
          </Route>
          <Route path="/quote">
            <Quote />
          </Route>
          <Route path="/calculator">
            <Calculator />
          </Route>
        </Switch>
      </div>
    </Router>
  );
}

export default App;
