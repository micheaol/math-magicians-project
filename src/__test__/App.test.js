import { screen, render, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import App from '../App';

afterEach(() => {
  cleanup();
});
describe('render App to the screen', () => {
  test('render App component to the screen', () => {
    render(<App />);
    const appElement = screen.getByTestId('app');
    expect(appElement).toBeInTheDocument();
  });

  test('matches snapshot', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
