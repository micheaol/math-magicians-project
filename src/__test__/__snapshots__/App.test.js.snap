// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App to the screen matches snapshot 1`] = `
<div
  data-testid="app"
>
  <nav
    data-testid="nav"
  >
    <h1>
      Math Magician
    </h1>
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        |
      </li>
      <li>
        <a
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        |
      </li>
      <li>
        <a
          href="/quote"
          onClick={[Function]}
        >
          Qoute
        </a>
      </li>
    </ul>
  </nav>
  <div
    className="home-wrapper"
    data-testid="home"
  >
    <h1>
      Welcome to our page!
    </h1>
    <div>
      <p>
        Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
      </p>
      <p>
        Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
      </p>
      <p>
        Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
      </p>
    </div>
    <div>
      <p>
        Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
      </p>
      <p>
        Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
      </p>
      <p>
        Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
      </p>
    </div>
    <div>
      <p>
        Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
      </p>
      <p>
        Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
      </p>
      <p>
        Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
      </p>
    </div>
  </div>
</div>
`;
